src/org/thoughtcrime/securesms/MediaPreviewActivity.java

```diff
@@ -407,8 +400,8 @@ private boolean isMediaInDb() {
     }
   }
 
-  public static boolean isContentTypeSupported(final String contentType) {
-    return contentType != null && (contentType.startsWith("image/") || contentType.startsWith("video/"));
+  public static boolean isContentTypeSupported(final Slide slide) {
```

Comment:
The name of this function does not make sense anymore, as it does not check MIME type. Maybe remove this function completely, as it is only used in one place anyway?

Reply:
It has two usages unfortunately:/ But I can rename it to `isTypeSupported`